subworkflows/local/input_check.nf

```diff
@@ -11,7 +11,7 @@ workflow INPUT_CHECK {
     main:
     SAMPLESHEET_CHECK ( samplesheet )
         .csv
-        .splitCsv ( header:true, sep:'\t' )
+        .splitCsv ( header:true, sep:',' )
```

Comment:
Might be good to double check this applies, if you are using the samplesheet (seems like a large change for a user as it's not doucmented)

Reply:
Hey @jfy133,
Thanks for the review, but I am unsure if I understand the size of this change since the comma separation only applies to the validated sample sheet and users still need to provide tab-separated input.
Looking forward to your output here.

Reply:
Hmm, I already understand the issue. 
I changed the separation of the sample sheet accordingly so that it is tab-separated 👍. 
Again, thank you for the feedback!

Reply:
Sorry original comment wasn't clear, reviewed at a rush , glad it made sense in the end!